README.md

```diff
@@ -1,13 +1,238 @@
 ![Kin Token](kin_android.png)
-# move-kin-android
-This repository contains modules that connect the ecosystem apps together.
+# Overview
+This repository contains a module which connects all the ecosystem apps together.
```

Comment:
reason I wrote 'modules' in plural is because in the future the one wallet module will be here too